### PR TITLE
[[ Community Docs ]] Clarifications to ; keyword

### PR DESCRIPTION
**Pull request by asayd**
- Wording change in summary.
- Example - variable declaration and improvements.
- References - changed line (keyword) to line (glossary).
- Fixed some embedded links.
- Hard wrapped long lines.
- Fixed formatting on embedded code block.

### CODE REVIEW

File: docs/dictionary/keyword/semi-colon.lcdoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ Type: keyword
 
 Syntax: ;
 
-Summary: The <character> <;> is used to place two <statement|statements> on a single <line>.
+Summary: The <character> <;> is used to place more than one <statement|statements> on a single <line>.
```
Comment (livecodeali) [inline docs/dictionary/keyword/semi-colon.lcdoc]: I think this should be `...more than one <statement> on a single <line>.`

Comment (asayd) [inline docs/dictionary/keyword/semi-colon.lcdoc]: @livecodeali Done. Good catch.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 1f176c9

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 1f176c9b77e14d918a21cb4e6a13768aa9391e49

**Comment by livecode-vulcan** [comment]
😎 test success 1f176c9b77e14d918a21cb4e6a13768aa9391e49
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2665)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2675)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2660)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2676)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2682)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2688)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2700)

**Pull request merged**